packages/react/src/components/QuilttButton.tsx

```diff
@@ -43,8 +51,15 @@ export const QuilttButton = <T extends AnyTag = 'button'>({
 
   const Button = as || 'button'
 
+  const handleClick = (event: MouseEvent<HTMLElement>) => {
+    // Call the user's onClick handler if provided
+    onClick?.(event)
```

Comment:
Let's be a bit more explicit
```suggestion
    if (onClick) onClick(event)
```